sapp/ui/schema.py

```diff
@@ -218,6 +218,14 @@ def resolve_issues(
             .where_issue_instance_id_is(issue_instance_id)
             .get()
         )
+
+        if len(issues) == 1:
+            other_issues = Instance(session, DBID(run_id)).get()
+            for other_issue in other_issues:
+                if other_issue == issues[0]:
+                    continue
+                issues[0].compute_and_update_similarity_with(other_issue)
```

Comment:
Can we move this in `compute_and_update_similarity_with`, and make it return the set directly? And I would rename it `similar_issues`.
Most users of `resolve_issues` won't need that, let's not make them pay for it.

Reply:
@arthaud do you mean to create a seperate endpoint for similar issues?

Reply:
I meant having a separate method on `IssueQueryResultType` that does all this (iterating on all issues and calling `compute_and_update_similarity_with`. There is no reason to have it here.

Reply:
Gotcha.